www/less/misc.less

```diff
@@ -47,3 +47,8 @@ a:hover {
     margin-bottom: 5px;
 }
 
+// WARNING! TEMPORARY FOR TESTING FORCE TOUCH. DO NO MERGE!
+* {
```

Comment:
hmmm, this could be a non-starter for this approach. if you're able to get the preview callbacks above to work, please re-enable this to see if you can still select text **and** peek on the page. if not, might need to go back to the drawing board